packages/eslint-config-4catalyzer/rules.js

```diff
@@ -19,6 +19,12 @@ module.exports = {
     'LabeledStatement',
     'WithStatement',
   ],
+  'no-underscore-dangle': ['error', {
+    allow: [],
+    allowAfterThis: true,
+    allowAfterSuper: false,
```

Comment:
does this make sense as well considering classes; e.g. private/protected

Reply:
Let's start with more conservative for now. I don't think we use that much inheritance anyway, and IMO things meant to be called by subclasses shouldn't be underscore-prefixed anyway, maybe.